### DISCUSSION TITLE
Converting Mac OS filenames to Windows

### DISCUSSION THREAD

**Discussion by hafdon** [BryanJenksCommunity/FAQ#84] [Q&A] [answered]
My first obsidian vault was on a mac os x. I have the vault linked to a git repo.

I have recently had to start using Windows 10. When I clone the repo locally, many of the filenames are incompatible (they include semicolons or other disallowed characters). 

Do you know of a solution to this problem (aside from running Obsidian from WSL2 linux distro, or changing all the filenames manually on the Mac and pushing to origin?

**Comment by tallguyjenks** [comment, marked as answer]
An easy way? on mac you can install the terminal file browser ranger and use that to rename all the files using vim which would make the renaming really easy. but that's just what I would do with my skillset, yet another thing that sucks about windows
